testapi/models.py

```diff
@@ -8,15 +8,14 @@
 from django.db import models
 from django.utils import timezone
 
-try:
+if settings.DATABASES['default']['ENGINE'] == 'django.db.backends.postgresql':
```

Comment:
changed the way we detect if we run test on postgresql or sqlite. previously it was with installed package, now it's with settings.